src/new-client/src/app/lib/types/index.tsx

```diff
@@ -126,8 +126,103 @@ type Session = {
   services: string[];
 };
 
+type FetchAvailabilityRequest = {
+  sites: string[];
+  service: string;
+  from: string;
+  until: string;
+  queryType: string;
```

Comment:
We could consider locking this down to `"NotSet" | "Days" | "Hours" | "Slots"` for extra type safety and/or readability